app/views/task_lists/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="TaskLists index">
+  <h2>Task Lists</h2>
+
+  <%= link_to("Add new task list", new_task_list_path, :class => 'action new') %>
+
+  <table class="listing">
+    <caption>Task list</caption>
+    <tr class="header">
+    </tr>
+    <% @task_lists.each do |list| %>
+      <%
+        if list.hidden
+          next
+        end
+      %>
+      <tr>
+        <td><%= list.name %></td>
+        <td><%= list.hidden ? 'Yes' : 'No' %></td>
```

Comment:
Try to keep views as simple as possible,
this is perfect candidate for helper method, something like this:

```ruby
module TaskListsHelper
  def hidden(list)
    list.hidden ? 'Yes' : 'No'
  end
end
```

Then in view just do:
`<td><%= hidden(list) %></td>`

Reply:
But I can see that in the code above you have `if list.hidden` which means this code will never be used